### ISSUE TITLE
Add localization support

### ISSUE THREAD

**Issue by AurusHuang** [dbgate/dbgate#347]
While a certain level of English is required for a database administrator to work, it is better to provide them interface with their native language, which can improve their efficiency when using DbGate. Please consider creating a cooperative translation project, and if such project exists, I'll be glad to help with Simplified Chinese translation (I live in Mainland China).

**Comment by janproch** [comment]
Thanks very much for suggestion and your offer, I will consider it.

I didn't suppose to make DbGate multi-language in past, but if there will be more upvotes for this feature, I could implement support for translations.

**Comment by kamust** [comment]
I agree.

**Comment by shuncon** [comment]
> Thanks very much for suggestion and your offer, I will consider it.
> 
> I didn't suppose to make DbGate multi-language in past, but if there will be more upvotes for this feature, I could implement support for translations.

Hello, I need Simplified Chinese, I hope to support, thank you

**Comment by kylin-kai** [comment]
I agree

**Comment by you7347** [comment]
I agree

**Comment by l2999019** [comment]
I agree

**Comment by ProjectInfinity** [comment]
Hi, please use the thumbs up emoji on the initial post to show support for this issue.

**Comment by shenxgan** [comment]
me too

**Comment by Zccccccccc9** [comment]
I agree.

**Comment by cotiga** [comment]
Would be cool

**Comment by TscCai** [comment]
me too, hope to support multi-language

**Comment by sdtm1016** [comment]
me too

**Comment by arc130** [comment]
I agree! Add plugin support for multiple languages can allow better leveraging of community support.

**Comment by you7347** [comment]
hope to support zh-cn

**Comment by youcaizhang** [comment]
I agree

**Comment by SikroYang** [comment]
I agree

**Comment by janproch** [comment]
The full localization will be longer process, but the core of the app is already localized

We use LLM models for machine translation to Chinese, if you find some error, please let us know

Current Chinese translation file is here:

https://github.com/dbgate/dbgate/blob/master/translations/zh.json

**Comment by you7347** [comment]
> The full localization will be longer process, but the core of the app is already localized
> 
> We use LLM models for machine translation to Chinese, if you find some error, please let us know
> 
> Current Chinese translation file is here:
> 
> https://github.com/dbgate/dbgate/blob/master/translations/zh.json

please tell me, how to use? thanks!
